### DISCUSSION TITLE
请问如何在多台虚拟机上实现联邦学习场景

### DISCUSSION THREAD

**Discussion by K2no** [secretflow/secretflow#118] [Q&A] [answered]
现有三台虚拟机，希望使用其中两台作为计算方（alice、bob），另一台作为结果方（charlie），来模拟联邦学习场景；官方文档中的介绍均为单机情况下模拟集群，请问如何在多台虚拟机上实现联邦学习场景

**Comment by ian-huu** [comment, marked as answer]
目前还没有文档说明，可以参考一下这个 issue：https://github.com/secretflow/secretflow/issues/50
